src/assets/content/dictionary/ubuntu.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Ubuntu
-definition: ''
+definition: "Ubuntu is a Linux distribution based on Debian and composed primarily of free and open-source software.Developed by the British company Canonical and a community of contributors under a meritocratic governance model"
```

Comment:
I suggested a small grammar change. Otherwise, looks good!
```suggestion
definition: "Ubuntu is a Linux distribution based on Debian and composed primarily of free and open-source software. It was developed by the British company Canonical and a community of contributors under a meritocratic governance model"
```